src/engraving/libmscore/barline.cpp

```diff
@@ -1040,7 +1040,7 @@ std::vector<PointF> BarLine::gripsPositions(const EditData& ed) const
     const PointF pp = pagePos();
 
     return {
-        PointF(lw * .5, y1 + bed->yoff1) + pp,
+        //PointF(lw * .5, y1 + bed->yoff1) + pp,
```

Comment:
removing redundant upper grip handle for Barline object, @Tantacrul was keen to test it like this before committing definitely (it's redundant as a grip handle but does at least provide visual feedback where the top of the barline is...)